modules/vba_documents/app/sidekiq/vba_documents/slack_status_notifier.rb

```diff
@@ -0,0 +1,144 @@
+
+      if percent_expired > EXPIRED_THRESHOLD
+        message_time = created_at_range.first.in_time_zone('America/New_York')
+                                       .strftime('%Y-%m-%d %I:%M:%S %p %Z')
+        fail_rate = ActiveSupport::NumberHelper.number_to_percentage(percent_expired, precision: 1)
+        message = "#{expired_count}(#{fail_rate}) " \
+                  "out of #{new_count} Benefits Intake uploads created since #{message_time} " \
+                  'have expired with no consumer uploads to S3' \
+                  "\nThis could indicate an S3 issue impacting consumers."
+
+        notify_slack(message, expired_details_rate_by_consumer(created_at_range))
+      end
+    rescue => e
+      notify_slack("'Expired' status notifier exception: #{e.class}", e.message)
+      raise e
+    end
+
+    def expired_details_rate_by_consumer(created_at_range)
+      # break out expired rates by consumer
+      consumer_all_counts = VBADocuments::UploadSubmission.where(created_at: created_at_range)
+                                                          .group(:consumer_name).count
+      consumer_exp_counts = VBADocuments::UploadSubmission.where(created_at: created_at_range, status: 'expired')
+                                                          .group(:consumer_name).count
+
+      # calc expired rate% by consumer
+      exp_rate = consumer_all_counts.map { |name, count| [name, (consumer_exp_counts[name].to_f / count) * 100] }
+
+      # sort by Consumer expired Rate, build slack reporting string
+      slack_details = "\n\t(Consumer, Expired Rate)\n"
+      exp_rate.sort_by { |e| -e[1] }.each do |cr|
+        slack_details += "\t#{cr[0]}: #{ActiveSupport::NumberHelper.number_to_percentage(cr[1], precision: 1)}\n"
+      end
+
+      slack_details
+    end
+
+    def report_uploaded
+      # Upload Submisions stuck in uploaded status indicate the consumers are able to push their
+      # file to s3 but we are unable to process it.
+      uss = VBADocuments::UploadSubmission.where(status: 'uploaded',
+                                                 created_at: ..UPLOADED_AGE_THRESHOLD_MINUTES.minutes.ago)
+                                          .order(:created_at)
+      if Flipper.enabled?(:decision_review_delay_evidence)
+        # if delay appeal evidence is on(max delay of 24 hours), look for any appeal's evidence
+        # uploads that have exceeded the max delay of 24 hours plus the normal time we allow
+        # the UploadScanner Sidekiq job to pickup the submission and upload to EMMS\CM
+        max_age = (24.hours + UPLOADED_AGE_THRESHOLD_MINUTES.minutes).ago
+        uss = uss.where.not(consumer_name: DELAYED_EVIDENCE_CONSUMERS)
+        uss = uss.or(VBADocuments::UploadSubmission.where(consumer_name: DELAYED_EVIDENCE_CONSUMERS,
+                                                          status: 'uploaded',
+                                                          created_at: ..max_age))
```

Comment:
Happy to see appeals evidence added!

Reply:
Since we added a max hold time of 24 hours for evidence uploads that automatically uploads these to Central Mail, it was pretty easy to have a go at catching these by adding 24 hours to the 100 minute "stuck" time